lib/checker_jobs/notifiers/logger.rb

```diff
@@ -0,0 +1,53 @@
+require 'logger'
+
+class CheckerJobs::Notifiers::Logger
+  DEFAULT = {
+    logdev: STDOUT,
+    level: Logger::INFO
+  }
+
+  def initialize(check, count, entries)
+    @check = check
+    @count = count
+    @entries = entries
+    raise CheckerJobs::InvalidNotifierOptions unless valid?
+
+    @logger = Logger.new(logdev)
+    @logger.level = level
+  end
+
+  def notify
+    @logger.add(level, format, @check.name.tr("_", " ").capitalize)
+  end
+
+  # override this
+  def format
+    "found #{@count} entries"
+  end
+
+  private
+
+  def valid?
+    (logdev.is_a?(String) || logdev.kind_of?(IO)) &&
+      [
+        Logger::UNKNOWN, Logger::FATAL, Logger::ERROR,
+        Logger::WARN, Logger::INFO, Logger::DEBUG
+      ].include?(level)
+  end
+
+  def level
+    @level ||= @check.klass.notifier_options[:level] ||
+      notifier_options[:level] ||
+      DEFAULT[:level]
+  end
+
+  def logdev
+    @logdev ||= @check.klass.notifier_options[:logdev] ||
+      notifier_options[:logdev] ||
+      DEFAULT[:logdev]
```

Comment:
Layout/MultilineOperationIndentation: Align the operands of an expression in an assignment spanning multiple lines.